package.json

```diff
@@ -118,11 +124,23 @@
     "start:proxy": "PROXY=true npm run start",
     "start:proxy:beta": "BETA=true npm run start:proxy",
     "travis:build": "NODE_ENV=production webpack --config config/test.webpack.config.js",
-    "travis:verify": "npm-run-all build:prod lint test",
-    "verify": "npm-run-all build lint test",
-    "nightly": "npm run travis:verify"
+    "travis:verify": "npm-run-all build:prod lint test:ct",
+    "verify": "npm-run-all build lint test:jest",
+    "nightly": "npm run travis:verify:ct",
+    "test:ct": "BABEL_ENV=componentTest cypress run --component",
+    "test:openct": "BABEL_ENV=componentTest cypress open --component",
+    "coverage": "bash coverage.sh"
   },
   "insights": {
     "appname": "remediations"
+  },
+  "nyc": {
+    "report-dir": "codecov-cypress",
```

Comment:
The same goes here:
```suggestion
    "report-dir": "cypress-coverage",
```

Reply:
Nice, catch. I was not really looking at the script as I copied it :)